_posts/newsletter/2020-04-09-this-month-in-solid.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: newsletter
+title: This Month in Solid 2020-04-09
+permalink: /newsletter/2020-04-09
+tags: [monthly, updates]
+categories: [Updates]
+---
+{% include twis-intro.md %}
+
+## Updates
+
+### [Events](https://solidproject.org/events)
+
+Solid World is an opportunity to meet people who are interested in or currently working on Solid.
+
+A big thank you to Travis Vachon, David Faveris, and Astrid Gamoneda for presenting at the first Solid World last Thursday. 
```

Comment:
last Thursday might need adjustment (unless the newsletter will be backdated)

Reply:
Is there a recording?

Reply:
Yep, see line 24. (And as mentioned in the other comment, indeed we're backdating.)